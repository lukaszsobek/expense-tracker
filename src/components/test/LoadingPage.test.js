import React from "react"
import { shallow } from "enzyme"

import { LoadingPage } from "../"

test("Renders loading screen", () => {
    const wrapper = shallow(<LoadingPage />)
    expect(wrapper).toMatchSnapshot()
})