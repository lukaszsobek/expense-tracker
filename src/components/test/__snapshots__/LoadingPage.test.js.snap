// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders loading screen 1`] = `
<div
  className="app-loading"
>
  <div
    className="app-loading__spinner"
  />
  Loading...
</div>
`;
